core/templates/dev/head/profile/Profile.js

```diff
@@ -29,22 +31,87 @@ oppia.controller('Profile', [
     $rootScope.loadingMessage = 'Loading';
     $http.get(profileDataUrl).success(function(data) {
       $rootScope.loadingMessage = '';
+      $scope.username = {
+        title: 'Username',
+        value: data.username,
+        helpText: (data.username)
```

Comment:
Use an ng-if on the popover, so that it's only displayed in the right circumstances. You'll probably want to compute a $scope.usernameIsLong variable here for that purpose.